packages/std/schema/cosmos_msg.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "CosmosMsg",
-  "anyOf": [
+  "oneOf": [
```

Comment:
Interesting change for a patch release. I'll pull this out to apply it to all the contracts as well.